packages/profile-sync-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/profile-sync-controller",
-  "version": "0.0.0",
+  "version": "1.0.0",
```

Comment:
Similar as for `notifications-controller`: new packages should start out at 0.0.0 until they are ready to be published. Otherwise, this one could get published accidentally:

``` suggestion
  "version": "0.0.0",
```

Reply:
addressed